Documentation/ReleaseNotes.md

```diff
@@ -0,0 +1,283 @@
+- Proximity lights now have per material color overrides and a "Subtractive Light" option to mimic proximity
+shadows
+- Clipping primitives can now be toggled on/off at runtime without requiring shader permutations to be known
+at (player) build time
+- Additive and transparent materials are now clipped correctly. A new property for non-opaque or cutout materials
+called “Blended Clipping Width” allows developers to set the width of the clip blend region
+- Lighting of back face triangles is now correct
+
+**Optimize Window**
+
+The MRTK [Optimize Window](Tools/OptimizeWindow.md) is a utility to help automate and inform in the process of
+configuring a mixed reality project for best performance in Unity.
+
+**Radial Solver**
+
+There have been improvements on vertical positioning. Check ‘Use Fixed Vertical Position’ to lock the vertical movement to achieve shell-style tag-along behavior. You can see the example of lazy-following tag-along behavior in the ‘ToggleFeaturesPanel’ prefab .
+
+**SceneSystem**
+
+MRTK 2.0.0 has added the [Scene System](SceneSystem/SceneSystemGettingStarted.md) to help with
+applications that contain more than once scene.
+
+**Service managers (experimental)**
+
+This release adds service managers to enable the light-weight addition of specific Microsoft 
+Mixed Reality Toolkit features, such as the Spatial Awareness system, individually.
+
+These service managers are imported as part of the Foundation package and are located in the
+MixedRealityToolkit.SDK\Experimental\Features folder and are a work in progress.
+
+Service manager prefabs are provided for the following services.
+
+- BoundarySystem
+- CameraSystem
+- DiagnosticsSystem
+- InputSystem
+- SpatialAwarenessSystem
+- TeleportSystem (requires the Input System)
+
+To use, drag and drop the desired prefab into your heirarchy and select the configuration
+profile. 
+
+> [!Note]
+> These service managers are currently experimental, may have issues and
+are subject to change. Please file any and all issues you encounter on GitHub
+
+**Slate**
+
+- Improved
+    - Usability of slates by adding the shadow based on the finger proximity
+    - ‘Follow Me’ behavior to match the HoloLens 2 shell behavior, using Radial Solver.
+- Fixed
+    - Border thickness issue fixed on flattend Bounding Box
+
+**SpatialObjectMeshObserver**
+
+We have added the SpatialObjectMeshObserver to improve developer productivity when working
+with the Spatial Awareness system. This observer reads mesh data from imported 3D models
+and uses them to simulate environmental data from devices such as Microsoft HoloLens 2.
+
+SpatialObjectMeshObserver is not enabled in the default profiles, please see the
+[Spatial Awareness Gettniving Started](SpatialAwareness/SpatialAwarenessGettingStarted.md) article
+for more information on configuring your application. 
+
+**System Keyboard**
+
+The system keyboard can now be used on all platforms. See the HandInteractionExamples scene in the Examples
```

Comment:
>The system keyboard can now be used on all platforms. [](start = 0, length = 53)

This is not true. It doesn't work on OpenVR, and that's because the underlying thing it uses is WMR specific. (i.e. the SystemKeyboard Unity API, which uses the underlying Windows InputPane API).

I would rather we don't have this section at all, as it gives people false hope that this is intended to be a cross-platform keyboard solution, when it isn't.

Reply:
removed